library/network/test/network_interfaces_test.rb

```diff
@@ -23,7 +23,11 @@ def mock_ppp
     .and_return "512\n"
 end
 
-describe Yast::NetworkInterfaces do
+# FIXME: find why it does not work. It looks like some problem with SCR agent
+# registration as .network.section is empty
+# Y2DEBUG mention "Agent registration: /home/abuild/rpmbuild/BUILD/yast2-4.2.4/library/network/src/scrconf last changed at Thu Jan  1 00:00:00 1970"
```

Comment:
The last_changed code is in fact unused.

Reply:
@jreidinger strace tells me you have removed /usr/share/YaST2/scrconf/network.scr

Reply:
@mvidner it is %check section, so it is not installed on system. Check is run before install. so it should use local https://github.com/yast/yast-yast2/blob/master/library/network/src/scrconf/network.scr.in which is probably that issue. WTF is that dynamic scr agent? I have to check how it works in past.